src/js/components/BreadcrumbComponent.jsx

```diff
@@ -76,8 +165,16 @@ var BreadcrumbComponent = React.createClass({
   },
 
   render: function () {
+    var classSet = classNames({
+      breadcrumb: true,
+      collapsed: this.state.collapsed
+    });
+
+    // Hacky, but ensures that the dimension measurements are up-to-date
+    _.defer(this.updateDimensions);
```

Comment:
This definitely needs another solution. React anti-pattern par excellence.

From https://facebook.github.io/react/docs/component-specs.html :
"
The render() function should be pure, meaning that it does not modify component state, it returns the same result each time it's invoked, and it does not read from or write to the DOM or otherwise interact with the browser (e.g., by using setTimeout).
"

 Is placing this in componentDidMount not working?